src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String nameMachine) {
+        super(nameMachine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getNameMachine() + " start work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getNameMachine() + " stoped work");
```

Comment:
There is a typo in the word 'stopped'. Additionally, the stop message should also be more informative. You could rephrase it to 'Bulldozer ' + getNameMachine() + ' stopped working.' to clearly indicate the type of machine and its state.